configuration/reissue-internal-certificates.adoc

```diff
@@ -24,19 +25,32 @@ include::modules/reissue-internal-certificates-scanner.adoc[leveloffset=+1]
 //Restart Scanner & Scanner DB
 include::modules/restart-scanner-and-scannerdb-containers.adoc[leveloffset=+2]
 
-[id="reissue-internal-certificates-secured-clusters"]
-== Reissuing internal certificates for Sensor, Collector, and Admission controller
-Sensor, Collector, and Admission controller use certificates to communicate with each other, and with Central.
+[id="reissue-internal-certificates-secured-clusters_{context}"]
+== Reissuing internal certificates for secured clusters
+Secured clusters contain the Collector, Sensor, Admission Control, and local Scanner components.
+These components communicate with each other, and with Central by using certificates.
 
-To replace the certificates, use one of the following methods:
+Choose the appropriate method to reissue the internal certificates:
 
-* Create, download, and install an init bundle on the secured cluster. You must have the `Admin` user role to create an init bundle.
-* Use the automatic upgrades functionality. Automatic upgrades are available only for static manifest deployments using the `roxctl` CLI.
-//Add link to automatic upgrades
+* Use the automatic certificate renewal feature. This is the recommended method for Operator and Helm deployments.
+* Create, download, and install an init bundle on the secured cluster. You must have the `Admin` user role to create an init bundle. This method is only recommended for Operator and Helm deployments if the certificates have already expired and the secured cluster can no longer connect to Central.
+* Use the automatic upgrades feature, which is only available for static manifest deployments by using the `roxctl` CLI. This method is only recommended if you have a specific installation requirement that necessitates the use of this method.
+
+//automatic certificate renewal
+include::modules/reissuing-internal-certificates-for-secured-clusters-by-using-automatic-certificate-renewal.adoc[leveloffset=+2]
+
+//Applying the latest internal certificates
+include::modules/applying-the-latest-internal-certificates.adoc[leveloffset=+3]
+
+//Verifying the status of automatic certificate renewal
+include::modules/verifying-the-status-of-automatic-certificate-renewal.adoc[leveloffset=+3]
```

Comment:
I would switch the order of these two, because first the user would check the status - and only apply the latest certificates if something is wrong.